src/adaptor/proxy/member.ts

```diff
@@ -0,0 +1,21 @@
+import type { Client, GuildMember } from 'discord.js';
+
+import type { Snowflake } from '../../model/id.js';
+import type { MemberResponseRunner } from '../../runner/member.js';
+import type { NewMember } from '../../service/welcome-message.js';
+
+type AllMemberModel = NewMember;
```

Comment:
```suggestion
import type { NewMember as AllMemberModel } from '../../service/welcome-message.js';
```
こう書くくらいなら別名で `import` したほうがよいかと

Reply:
Fixed https://github.com/approvers/OreOreBot2/pull/1028/commits/17237327fa8e50c853f9f96424520ad06fc67317